### ISSUE TITLE
Docs feedback: /http_api/snapshot.md - Make clear that this API endpoint is NOT for use by customer, but only for use by the Grafana UI frontend

### ISSUE THREAD

**Issue by ethirolle** [grafana/grafana#49762]
I had a customer who wasted hours trying to figure out how to make a call to this Snapshot API endpoint. It turns out that this API endpoint is really only for use by the Grafana UI. We should make this clear in the docs, so other Grafana users do not waste their time with this.

**Details**
In my recent Grafana versions, I have enabled the "Swagger UI" beta feature, which allows me to see a full API specification (and API testing UI) in the Grafana instance itself (at <grafana-server-url>/swagger-ui). When I look at the info for the snapshot POST API endpoint, it has this description:
 
![image](https://user-images.githubusercontent.com/10763196/170776183-3a82b7c0-088c-45c9-afc2-10fc6d94f327.png)
 
Here is that text:

> When creating a snapshot using the API, you have to provide the full dashboard payload including the snapshot data. This endpoint is designed for the Grafana UI.
 
In other words: this API endpoint really may not be intended to work except from the Grafana UI (which hits this API endpoint when creating a snapshot). The trick is that your API payload needs to include BOTH the dashboard JSON and the snapshot data.

**Fix for docs**
I suggest we add some language to this doc page that is similar to the language used in our Swagger UI above. And perhaps add another sentence to make it doubly clear that this API endpoint cannot really be used from outside the Grafana UI.

Thanks!

**Comment by chri2547** [comment]
Closing this issue because it's over a year old, in an unresourced area, and not likely to be worked on in the next year.
